galite-core/src/main/kotlin/org/kopi/galite/util/Fax.kt

```diff
@@ -0,0 +1,770 @@
+      fax.command("JKILL $job")
+      Utils.log("Fax", "Kill 1: $job")
+      fax.endCon()
+    }
+
+    /**
+     * Convenience method
+     */
+    fun clearJob(host: String,
+                 user: String,
+                 job: String) {
+      clearJob(HFAX_PORT, host, user, job)
+    }
+
+    /**
+     * Convenience method
+     */
+    fun clearJob(port: Int,
+                 host: String,
+                 user: String,
+                 job: String) {
+      val fax = Fax(port, host)
+      fax.login(DEFAULT_USER) // !!! laurent 20020626 : why DEFAULT_USER and not user ?
+      fax.command("JDELE $job")
+      Utils.log("Fax", "Delete 1: $job")
+      fax.endCon()
+    }
+
+    /*
+   * ----------------------------------------------------------------------
+   * HANDLE THE QUEUES --- ALL QUEUES ARE HANDLED BY THAT METHOD
+   * ----------------------------------------------------------------------
+   */
+       private fun getQueue(port: Int, host: String, user: String, qname: String): String {
+      val fax = Fax(port, host)
+      val ret: String // Hole Statusinformationen
+      fax.login(user)
+      fax.command("IDLE 900")
+      fax.command("TZONE LOCAL")
+      fax.command("JOBFMT \" %j| %J| %o| %e| %a| %P| %D| %s\"")
+      fax.command("RCVFMT \" %f| %t| %s| %p| %h| %e\"")
+      fax.command("MDMFMT \"Modem %m (%n): %s\"")
+      ret = fax.infoS(qname)
+      fax.endCon()
+      return ret
+    }
+
+    // ----------------------------------------------------------------------
+    // IMPLEMENTATION
+    // ----------------------------------------------------------------------
+    /*
+   * ----------------------------------------------------------------------
+   * READS ANY QUEUE
+   * RETURNS A VECTOR OF STRINGS
+   * ----------------------------------------------------------------------
+   */
+    private fun readQueue(host: String, user: String, qname: String): Vector<FaxStatus> {
+      val queue: Vector<FaxStatus> = Vector<FaxStatus>()
+      try {
+        val ret = getQueue(HFAX_PORT, host, user, qname)
+        val token = StringTokenizer(ret, "\n")
+        Utils.log("Fax", "READ $qname : host $host / user $user")
+        while (token.hasMoreElements()) {
+          try {
+            val str = token.nextElement().toString()
+            val prozess = StringTokenizer(str, "|")
+            if (qname != "recvq") {
+              queue.addElement(FaxStatus(prozess.nextToken().trim { it <= ' ' },  // ID
+                      prozess.nextToken().trim { it <= ' ' },  // TAG
+                      prozess.nextToken().trim { it <= ' ' },  // USER
+                      prozess.nextToken().trim { it <= ' ' },  // DIALNO
+                      prozess.nextToken().trim { it <= ' ' },  // STATE (CODE)
+                      prozess.nextToken().trim { it <= ' ' },  // PAGES
+                      prozess.nextToken().trim { it <= ' ' },  // DIALS
+                      prozess.nextToken().trim { it <= ' ' })) // STATE (TEXT)
```

Comment:
Remove ( ) from comment : 
// State text
Or it will be considered as commented code in sonarLint